arithmetization/src/main/java/net/consensys/linea/zktracer/module/romlex/RomLex.java

```diff
@@ -62,6 +60,7 @@ public class RomLex
       new ModuleOperationStackedSet<>();
 
   @Getter private List<RomOperation> sortedOperations;
+  Map<ContractMetadata, Integer> precomputedNbOfOperationsPerMetadata = new HashMap<>();
```

Comment:
could be final + I don't understand at all the name, it should be something like `cfiMetadataCorrespondance`